src/components/about/ValueCard.tsx

```diff
@@ -1,19 +1,29 @@
+"use client";
+import { motion } from "framer-motion";
```

Comment:
import should be from motion/react
```suggestion
import { motion } from "motion/react";
```